t/errlog.t

```diff
@@ -932,3 +932,65 @@ maybe log lines: #1
 end
 \z
 --- skip_nginx: 2: <1.11.2
+
+
+
+=== TEST 23: the system default filter level is "debug"
```

Comment:
@spacewander I think we need some test cases for different phases. 

for example `init_by_lua*`, need to cover https://github.com/openresty/lua-nginx-module/pull/1079/files#diff-7caeff0fc42c25ea398bef6b2257cfc7R419